src/styles/navbar.css

```diff
@@ -0,0 +1,139 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600&display=swap');
+/* My Component Library link  */
+@import url("https://rebalance-ui.netlify.app/all_component.css");
+
+.home {
+    background-color: var(--bg-color);
+    padding: 13rem 6%;
+}
+
+section {
+    padding: 6rem 6%;
+}
+
+.heading {
+    text-align: center;
+    margin-bottom: 2rem;
+    position: relative;
+}
+
+.heading::before {
+    content: '';
+    position: absolute;
+    top: 50%;
+    left: 0;
+    transform: translateY(-50%);
+    width: 100%;
+    height: .01rem;
+    background: rgba(0, 0, 0, .1);
+    z-index: -1;
+}
+
+.heading span {
+    font-size: 3rem;
+    padding: .5rem 2rem;
+    color: var(--black);
+    background: var(--white-hover);
+    border: 2px solid var(--secondary-gray);
+    border-radius: 6px;
+    cursor: pointer;
+}
+
+.heading span:hover {
+    border-color: var(--yellow-color);
+}
+
+.btn {
```

Comment:
You could have used a more specific class name here. 
Rest everything is good to merge

Reply:
okay